### PR TITLE
queue: Another attempt at cube map rendering

### PR DESCRIPTION
**Pull request by magcius**
**Connections**
_Link to the issues addressed by this PR, or dependent PRs in other repositories_
Issue #2139

**Description**
_Describe what problem this is solving, and how it's solved._

The last OR was wrong, because I misread a (confusing) line in the GL ES spec.

**Testing**
_Explain how this change is tested._

It wasn't, this is a blind PR. Please don't merge until @superdump has been given an opportunity to test.

### CODE REVIEW

**Comment by superdump** [comment]
Confirmed working for the point light cube map shadow map texture as in #2138 in both Chrome and Firefox. Thanks!

File: wgpu-hal/src/gles/queue.rs [hunk 1]
```diff
@@ -94,6 +91,14 @@ impl super::Queue {
                         view.mip_levels.start as i32,
                         view.array_layers.start as i32,
                     );
+                } else if target == glow::TEXTURE_CUBE_MAP {
+                    gl.framebuffer_texture_2d(
```
Comment (kvark) [inline wgpu-hal/src/gles/queue.rs:95]: nit: could be merged more closely with the other `framebuffer_texture_2d`

Comment (magcius) [reply wgpu-hal/src/gles/queue.rs:95]: i figured it was clearer this way than doing `if target == glow::TEXTURE_CUBE_MAP { texture = CUBE_MAP_FACES[...]; }` but feel free to make your own judgment

**Comment by kvark** [comment]
Published in wgpu-hal-0.11.5

**Pull request merged**